composer.json

```diff
@@ -31,7 +31,10 @@
         "symfony-cmf/testing": "~1.2",
         "sulu/document-manager": "dev-develop",
         "sulu/test-bundle": "dev-develop",
-        "massive/search-bundle": "0.7.*"
+        "massive/search-bundle": "dev-develop",
+        "pulse00/ffmpeg-bundle": "~0.5.2",
+        "php-ffmpeg/php-ffmpeg": "0.5.*",
```

Comment:
they 'have been" a dependency by sulu/sulu: "dev-develop"
"composer update" was not possible without that